shared/middleware/channels_api.rb

```diff
@@ -164,7 +164,7 @@ class ChannelsApi < Sinatra::Base
   # Marks the specified channel as published.
   #
   post %r{/v3/channels/([^/]+)/publish/([^/]+)} do |channel_id, project_type|
-    not_authorized if under_13?
+    not_authorized if under_13? && !%w(artist playlab).include?(project_type)
```

Comment:
> I don't know much about our 'publish' action. Is it sufficient to trust the passed project_type here? What happens if a student under 13 publishes a channel with an applab project but passes artist as the type?

Very creative, Brad! If a young student hacker passes in the wrong project type using `curl`, they could effectively publish their applab project as an artist or playlab project. the source code wouldn't show up, but the thumbnail image could. We could address this, however it seems like way more trouble than lying about their age in the first place.

Similarly, someone could publish a weblab app as an frankenstein-applab app, but I don't see any danger there besides potentially breaking the app in the process.

In the long run, once all apps already know what project type they are, we should be able to omit this parameter from the `publish` request. If we think there is any lingering problem here, then we could prioritize removing this parameter after the backfill.